src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1873,4 +1874,31 @@ OrbitMainWindow::DownloadFileFromInstance(std::filesystem::path path_on_instance
 
   return service_deploy_manager->CopyFileToLocal(std::move(path_on_instance), std::move(local_path),
                                                  std::move(stop_token));
+}
+
+orbit_base::CanceledOr<void> OrbitMainWindow::DisplayStopDownloadDialog(
+    const orbit_client_data::ModuleData* module) {
+  using orbit_config_widgets::StopSymbolDownloadDialog;
+  using Result = StopSymbolDownloadDialog::Result;
+
+  StopSymbolDownloadDialog dialog{module};
+  Result dialog_result = dialog.Exec();
+
+  orbit_base::CanceledOr<void> return_canceled_or;
+  switch (dialog_result) {
+    case orbit_config_widgets::StopSymbolDownloadDialog::Result::kCancel:
```

Comment:
Maybe just use`Result::kCancel` as we have the above using statement.

Reply:
yes :) done